script/hypercube/hypercube.sh

```diff
@@ -416,8 +416,13 @@ function install_vpnclient() {
 function install_hotspot() {
   logfile ${FUNCNAME[0]}
 
-  yunohost app install hotspot --force --debug\
-    --args "domain=$(urlencode "${settings[yunohost,domain]}")&path=/wifiadmin&wifi_ssid=$(urlencode "${settings[hotspot,wifi_ssid]}")&wifi_passphrase=$(urlencode "${settings[hotspot,wifi_passphrase]}")&firmware_nonfree=$(urlencode "${settings[hotspot,firmware_nonfree]}")" |& logfilter
+  if [ "${settings[hotspot,do_not_install]}" == yes ]; then
+    touch "${log_filepath}/do_not_install_hotspot"
```

Comment:
I think it would be nice to have a boolean setting called `enabled` for each service in the hypercube file. So this is a great step :)

Here I would suggest:
```suggestion
  if [[ "${settings[hotspot,enabled]} == false ]]; then
    touch "${log_filepath}/hotspot_disabled"
```

Also, in your hypercube file, you would then have a boolean instead of "yes/no" string :
```json
"hotspot": {
    "enabled": false,
    ...
}
```

Reply:
Your suggestion makes this part more understandable. Thank for checking !